config/pipelines.yml

```diff
@@ -102,6 +102,10 @@ default: &default
                 tag_set_name:
                   type: :model
                   value: tag.tag_set_name
+                pipeline_id_lims:
```

Comment:
Not convinced by the name 'pipeline_id_lims' for Libary type in the messsage as it suggests that pipeline to library type is 1:1. Can it not be 'library_type'?

Reply:
pipeline_id_lims is the name of the column in the warehouse so we can't change it.

Reply:
Ok

Reply:
Well the column in the warehouse doesn't exist yet, so could be renamed; but this name was picked for consistency with the iseq_flowcell table and was originally specified by NPG (I can't quite remember why they didn't want library_type).

I have highlighted the column name in an email for Carol S, so there might still be a request to change it.